edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -40,29 +41,52 @@ class EnsureJWTAuthSettingsMiddleware(MiddlewareMixin):
     """
     _required_permission_classes = (NotJwtRestrictedApplication,)
 
+    @staticmethod
+    def _get_decomposed_permissions(permission):
+        """
+        Given a composed permission class or instance, return it component parts.
```

Comment:
There's a typo here:

```suggestion
        Given a composed permission class or instance, return its component parts.
```